pose_calculator.py

```diff
@@ -137,15 +138,15 @@ def get_values(self, display=True):
 
         if len(contours) < 1:
             self.display_windows(frame, mask)
-            return [360, 360, 360], -1
+            return [float('nan'), float('nan'), float('nan')], [float('nan'), float('nan'), float('nan')]
```

Comment:
It's best to use None if you don't want to return anything. NaN behaves in interesting ways that can be difficult to debug.